database.py

```diff
@@ -30,7 +30,7 @@ def open(self, db_path: str):
 		c = self.db.execute('''PRAGMA user_version''')
 		user_version = c.fetchone()[0]
 
-		if user_version < 2:
+		if user_version < 2 and user_version >0:

```

Comment:
sorry to make you wait so long. please format as such:
```suggestion
		if user_version < 2 and user_version > 0:
```

Reply:
fixed formating